doc/tutorial/basic/tutorial.rst

```diff
@@ -5,7 +5,7 @@ An introduction to machine learning with scikit-learn
 
 .. topic:: Section contents
 
-    In this section, we introduce the `machine learning 
+    In this section, we introduce the `machine learning
```

Comment:
In general we try to avoid cosmetic changes like these because of the merge-conflicts they might create.
It's fine for now, though.